### PR TITLE
docs: standardize gitlab documentation

### PR DESCRIPTION
**Pull request by bbodenmiller**
- Swap scopes order to match GitLab UI
- Update casing to match https://github.com/GitCredentialManager/git-credential-manager/blob/main/docs/configuration.md

### CODE REVIEW

**Comment by Marodero56** [comment]
> * Swap scopes order to match GitLab UI
> * Update casing to match https://github.com/GitCredentialManager/git-credential-manager/blob/main/docs/configuration.md

**Review by influ3 — COMMENTED** [review]
GreatGreat

**Comment by AliTahirGujjar** [comment]
Enhancing

**Review by AliTahirGujjar — COMMENTED** [review]
Sa

**Review by influ3 — COMMENTED** [review]
Yes

**Review by influ3 — COMMENTED** [review]
Eyyy

File: docs/gitlab.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
+'write_repository' scopes.
 1. Copy the application ID and configure
-`git config --global credential.https://gitlab.example.com.GitLabDevClientId <APPLICATION_ID>`
+`git config --global credential.https://gitlab.example.com.gitLabDevClientId <APPLICATION_ID>`
 1. Copy the application secret and configure
-`git config --global credential.https://gitlab.example.com.GitLabDevClientSecret
+`git config --global credential.https://gitlab.example.com.gitLabDevClientSecret
 <APPLICATION_SECRET>`
-1. Configure authentication modes to include 'browser'
+1. Optional if you want to force OAuth:
```
Comment (ldennington) [inline docs/gitlab.md:19]:
```suggestion
1. If you would like to use OAuth, configure authentication modes to include 'browser'
```

**Review by ldennington — APPROVED** [review]
Thanks for working to improve our docs!

File: docs/gitlab.md (hunk 1 above)
Comment (bbodenmiller) [reply docs/gitlab.md:19]: @ldennington I don't think that wording quite works. In my experimentation if that option is provided only browser auth is authorized whereas if `browser` option is not provided user is prompted to select between browser, token, and username/password.

Comment (ldennington) [reply docs/gitlab.md:19]: Perhaps then we could say something like `Optional if you want to force browser auth`?

Comment (bbodenmiller) [reply docs/gitlab.md]: Updated

**Pull request merged**